babi/screen.py

```diff
@@ -225,11 +225,15 @@ def _get_string(self, wch: str) -> str:
     def _get_char(self) -> Key:
         if self._buffered_input is not None:
             wch, self._buffered_input = self._buffered_input, None
-        else:
```

Comment:
this still needs to be in an else block otherwise it's going to read multiple things when things get buffered

I wonder if we should set a timeout on this loop as well 🤔

Reply:
Okay I think I understand. I'll try to work on this more today! :)